### PR TITLE
Might fix memes.

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Fixed a few issues with memetic anomalies, specifically #602.

Can't test it as I'm on vacation and I need another person to test it with.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
Wow, this is really old code. Didn't even see that.

File: code/modules/mob/living/parasite/meme.dm [hunk 1]
```diff
@@ -21,10 +21,9 @@ var/controlling
 /mob/living/carbon/var/list/parasites = list()
 
 /mob/living/parasite
-	var
-		mob/living/carbon/host // the host that this parasite occupies
+	var/mob/living/carbon/host // the host that this parasite occupies
 
-	Login()
+/mob/living/parasite/Login()
```
Comment (skull132) [inline code/modules/mob/living/parasite/meme.dm]: Indent the contents of this block back 1, they're breaking the compiler.

File: code/modules/mob/living/parasite/meme.dm [hunk 2]
```diff
@@ -82,16 +80,16 @@ var/controlling
...
 		return
 	// a host without brain is no good
 	if(!host.mind)
-		src << "\red <b>Your host has no mind.. you fade away..</b>"
+		src << span("danger", "Your host has no mind... you fade away...")
 		src.death()
 		return
 	if(host.stat == 2)
-		src << "\red <b>Your host has died.. you fade away..</b>"
+		src << span("danger", "Your host has died... you fade away..."
```
Comment (skull132) [inline code/modules/mob/living/parasite/meme.dm]: Missing closing `)`.

File: code/modules/mob/living/parasite/meme.dm [hunk 3]
```diff
@@ -151,10 +149,10 @@ var/controlling
 // Try to use amount points, return 1 if successful
 /mob/living/parasite/meme/proc/use_points(amount)
 	if(dormant)
-		usr << "\red You're dormant!"
+		usr << span("warning", "You're dormant!")
 		return
 	if(src.meme_points < amount)
-		src << "<b>* You don't have enough meme points(need [amount]).</b>"
+		src << span("notice", "You don't have enough meme points (need [amount]).)"
```
Comment (skull132) [inline code/modules/mob/living/parasite/meme.dm]: Swap the last `)"` for `")`

**Comment by Lohikar** [comment]
Closing due to inactivity.

**Pull request closed without merging**